### ISSUE TITLE
Misleading Scenario statistics when Background Steps failed

### ISSUE THREAD

**Issue by florianrein** [damianszczepanik/cucumber-reporting#1218]
When I execute a feature containing 1 Scenario, where a Background Step fails, the Background section is marked as red as expected, but the following scenario is reported as green/passed. In the statistics the number of Scenarios failed is 0 albeit the number of failed Steps being correctly displayed as 1. Please see the attached screenshots and the link to my example project down below.

<img width="1391" alt="Bildschirmfoto 2024-11-20 um 19 03 56" src="https://github.com/user-attachments/assets/9314546b-16b7-40ad-b58e-9e0d3c659f83">
<img width="1424" alt="Bildschirmfoto 2024-11-20 um 19 04 53" src="https://github.com/user-attachments/assets/952c4609-47c4-46ac-b602-6ba58f1e8796">

https://github.com/florianrein/masterthought-cukes-report-min-example

**Comment by damianszczepanik** [comment]
That might be intentional. Attach JSON file for verification.

**Comment by florianrein** [comment]
json file, here you go. :) 
[test-report.json](https://github.com/user-attachments/files/17846384/test-report.json)
Intentional would be ugh. Can I rely on the constellation "Steps 1 RED, Scenario Green => Background Problem" in that case?
Asking for e.g. automated evaluation.

**Comment by damianszczepanik** [comment]
In attached file one step is passing, one failing and one is skipped and this is what the chart is presenting

**Comment by florianrein** [comment]
@damianszczepanik Yes, the Step Section is fine, but marking the Scenario as passed is not really true. It did not run. I'd expect a skipped (blue) or failed (red) scenario, but if this is intentional, then at least I have a statement for my peers.

**Comment by damianszczepanik** [comment]
This is report generated for attached JSON 
![obraz](https://github.com/user-attachments/assets/bdf80b06-f58a-43f0-85d7-463de5c3a83d)
and IMO this is correct

**Comment by florianrein** [comment]
@damianszczepanik Hö?? Okay.. this is entirely different from the report I got as shown in the screenshot.. and as I'd expect it. Did you use a different cucumber-reporting version? What does Qualifier: attachments mean? These are the main differences I could think of here.

**Comment by damianszczepanik** [comment]
This is the output from recent version
